tests/scripts/thread-cert/border_router/test_on_link_prefix.py

```diff
@@ -188,10 +188,10 @@ def test(self):
 
         host_on_link_addr = host.get_matched_ula_addresses(ON_LINK_PREFIX)[0]
 
-        # Wait 30 seconds for the radvd `ON_LINK_PREFIX` to be invalidated
-        # and make sure that Thread devices in both networks can't reach
-        # the on-link address.
-        self.simulator.go(30)  # Valid Lifetime of radvd PIO is set to 60 seconds.
+        # Wait for the radvd `ON_LINK_PREFIX` to be invalidated and make sure
+        # that Thread devices in both networks can't reach the on-link address.
+
+        self.simulator.go(1802)  # Valid Lifetime of radvd PIO is set to 1800
```

Comment:
This step takes 30 minutes which makes the 'border router' workflow take 50 minutes (used to be ~30minutes). Is there a way to make it spend less time? Maybe we can make the min preferred lifetime configurable via CLI but that is probably an over kill.

Reply:
I expected these tests to be running using virtual time. I believe all  certification tests scrips are set up this way.  If that's not currently the case for `borderrouter` related tests, then I agree that a 30-minute wait is undesirable.

Considering the discovered prefix timeout behavior is already well-covered in the faster `test_routing_manager` unit test,  it seems reasonable to remove this step from the script. I've included this change in a new push.

Reply:
SGTM. Thanks!